server_tests/test/clientTests.js

```diff
@@ -687,39 +686,39 @@ describe("RUST API TEST SUITE", function() {
             this.timeout(timeoutValue);
             return testClient.takePicture(sessionId)
             .then(function onSuccess (res) {
-                Comparison.oscTakePictureOutput(res);
+                validate.done(res.body, schema.names.commandTakePicture);
                 fileUri = res.body.results.fileUri;
                 return testClient.getImage(fileUri);
             })
-            .then((res) => Comparison.oscGetImageOutput(res))
+            .then((res) => validate.done(res.body, schema.names.commandGetImage))
```

Comment:
This is not going to be correct because getImage actually returns the image data itself, not an OSC command response.